src/components/Map/IpcMap/IpcChoropleth.tsx

```diff
@@ -26,6 +28,22 @@ function IpcChoropleth({
         setSelectedCountryId={setSelectedCountryId}
         selectedCountryId={selectedCountryId}
       />
+      {!ipcRegionData && selectedCountryId && (
+        <GeoJSON
```

Comment:
Maybe you could extract this into a CountrySkeleton component and pass data and color there?